api/lib/source/cosmosV0-source.js

```diff
@@ -288,6 +290,66 @@ class CosmosV0API extends RESTDataSource {
     )
   }
 
+  async getRecentProposals() {
+    // while we don't have proposals in DB this is the only way
+    const proposals = await this.getAllProposals()
+    return proposals.sort(
+      (a, b) => new Date(b.creationTime) - new Date(a.creationTime)
+    )
+  }
+
+  async getTopVoters() {
+    // for now defaulting to pick the 5 largest voting powers
+    return _.take(
+      _.reverse(
+        _.sortBy(this.store.validators, [
+          (validator) => {
+            return validator.votingPower
+          }
+        ])
+      ),
+      5
+    ).map(({ operatorAddress }) => operatorAddress)
+  }
+
+  async getGovernanceOverview() {
```

Comment:
should all these data points be in one function?

Reply:
I think it is fine like this, since they are just called as a bunch here

Reply:
@jbibla can you clarify what you mean?

Reply:
this one function is making 2 requests and calling two other functions - should one function make one request instead?

Reply:
I don't see an issue here, if I am getting right what you mean. Why is a bad idea to have a function call many different ones? (in this case actually 5 or 6) I guess I am not getting straight what you mean

Reply:
let'S remove recent proposals from here (see above)

Reply:
where is above exactly? Please link :pray:

Reply:
ah ok, found it: https://github.com/luniehq/lunie/pull/4607#discussion_r469638434

It was actually "below" :stuck_out_tongue: